Ok, I just tried your patch, and I get the following error (after I did `pip install vunit_hdl`):

```
mike@mike-desktop2 ~/git/Ruskuls/HyperRAM/simulation-vunit (vunit-support=) $ ./run.py 
modelsim
WARNING - Entity tb_clk matches testbench name regex ^(tb_.*)|(.*_tb)$ but has no generic runner_cfg and will therefore not be run.
in file /home/mike/git/Ruskuls/HyperRAM/simulation-vunit/test/tb_clk.vhd
Traceback (most recent call last):
  File "/home/mike/.local/lib/python3.10/site-packages/vunit/ui/__init__.py", line 735, in main
    all_ok = self._main(post_run)
  File "/home/mike/.local/lib/python3.10/site-packages/vunit/ui/__init__.py", line 779, in _main
    all_ok = self._main_run(post_run)
  File "/home/mike/.local/lib/python3.10/site-packages/vunit/ui/__init__.py", line 806, in _main_run
    self._compile(simulator_if)
  File "/home/mike/.local/lib/python3.10/site-packages/vunit/ui/__init__.py", line 950, in _compile
    simulator_if.compile_project(
  File "/home/mike/.local/lib/python3.10/site-packages/vunit/sim_if/__init__.py", line 209, in compile_project
    self.add_simulator_specific(project)
  File "/home/mike/.local/lib/python3.10/site-packages/vunit/sim_if/modelsim.py", line 118, in add_simulator_specific
    mapped_libraries = self._get_mapped_libraries()
  File "/home/mike/.local/lib/python3.10/site-packages/vunit/sim_if/modelsim.py", line 225, in _get_mapped_libraries
    cfg = parse_modelsimini(self._sim_cfg_file_name)
  File "/home/mike/.local/lib/python3.10/site-packages/vunit/sim_if/modelsim.py", line 426, in parse_modelsimini
    cfg.read_file(fptr)
  File "/usr/lib/python3.10/configparser.py", line 719, in read_file
    self._read(f, source)
  File "/usr/lib/python3.10/configparser.py", line 1117, in _read
    raise e
configparser.ParsingError: Source contains parsing errors: '/home/mike/git/Ruskuls/HyperRAM/simulation-vunit/vunit_out/modelsim/modelsim.ini'
	[line 1970]: 'l   arraymode    - consider rand_mode of unpacked array field independently from its elements\n'
```

I don't have any experience with VUnit, so I'm unsure how to debug/proceed here. It seems like it is selecting the wrong top-level testbench, maybe? Could you please help me?

BTW: I really appreciate you making this pull request and adding support for VUnit. This is a great improvement!! Now I just need to get it to work on my machine ...

--- a/simulation-vunit/README.md
+++ b/simulation-vunit/README.md
@@ -1,0 +1,23 @@
+# Simulation
+
+This folder contains files necessary to run an RTL simulation of the HyperRAM
+implementation with Vunit simulator.
+
+## Setting up Vunit
+
+software prerequisites:
+- python 3.6 or higher
+- python module - [`vunit-hdl`](https://vunit.github.io/).
+- `Modelsim`. **Note! Quartus and Lattice development tools are shipped with free version**
+
+### Configuration Vunit in Ubuntu
+- Add environment variables to your system as below (edit `bashrc`)
+- `export VUNIT_MODELSIM_PATH="/tools/lscc/radiant/2022.1/modeltech/linuxloem/"` - path to Modelsim in your machine.
+- `export VUNIT_SIMULATOR="modelsim"` - configure `Modlesim` as Vunit simulator. 
+
+## Running simulation
+Open `simulation-vunit` folder and type:
+- `python run.py` - to run all available tests
+- `python run.py --list` - list all available tests
+- `python run.py tb_lib.hyperram_tb.bulk_write_bulk_read_operation` - run one specified test
+- `python run.py --gui` - open and run tests in GUI.

